.github/workflows/ci-pr-build.yaml

```
@@ -26,6 +26,8 @@ jobs:
         include:
           - chartDir: alerts/charts
             chartName: alerts
+          - chartDir: audit-logs/charts
+            chartName: audit-logs-operator
```

NIT: I dont see this variable being used anyways, but this way it is in sync with the `helm-release.yaml`
```suggestion
            chartName: audit-logs
```